go.mod

```diff
@@ -3,7 +3,6 @@ module github.com/IBM/netaddr
 go 1.16
 
 require (
-	github.com/davecgh/go-spew v1.1.0 // indirect
-	github.com/pmezard/go-difflib v1.0.0 // indirect
-	github.com/stretchr/testify v1.2.1
+	github.com/ecbaldwin/trie v1.0.0
+	github.com/stretchr/testify v1.7.0
```

Comment:
This is what `go mod get` + `go mod tidy` did when I added the trie package.

Reply:
NOTE that before we merge this, I need to tag v1.2.0 in the trie repository and update this to pull that in by version.